lab/changelog.md

```diff
@@ -4,6 +4,56 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Lab v2.5.0 - <TODO: Add the release date>
+
+## What's New
+
+* In case of multiple Cypher queries in the query editor, you will be able to select a single query and run
+  just that one without a need to comment out all the other Cypher queries.
+
+<TODO: Add screenshot here>
+
+* A limit of having one query execution view has been removed and you can have multiple query executions now. It makes
+  life easier especially when you want to compare different Cypher queries side to side in terms of speed or
+  query results.
+
+<TODO: Add screenshot here>
+
+* Showing only the last result when running multiple Cypher queries at once has been changed. Now, results for each
+  Cypher query will be returned and you will be able to select and view each one.
```

Comment:
```suggestion
* If you are running multiple Cypher queries at once, you can now view the result of each one instead of viewing just the last result. 
```